modules/pytket-qiskit/docs/api.rst

```diff
@@ -13,19 +13,16 @@ API documentation
     :special-members: __init__
     :inherited-members:
     :members:
-    :exclude-members: get_state, get_unitary
```

Comment:
Aren't we getting rid of these?

Reply:
Might be my misunderstanding: I thought this was stating that `get_state` and `get_unitary` should be exlucded from the docs -- given that they don't exist anymore, I thought excluding them would be redundant.

Reply:
Sorry, me being dumb.

Reply:
Just double checked the docs. I think my understanding is correct: if we don't have these methods anymore, then this `:exclude-members:` statement should be deleted, right?

Reply:
EDIT: somehow your comment didn't appear at first